pkg/report/writer.go

```diff
@@ -49,7 +48,6 @@ func Write(ctx context.Context, report types.Report, option flag.Options) (err e
 			Output:               output,
 			Severities:           option.Severities,
 			Tree:                 option.DependencyTree,
-			ShowMessageOnce:      &sync.Once{},
```

Comment:
Note to reviewers: this field has been removed because it has not been used elsewhere